spec/factories/scheme.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :scheme do
-    service_name { Faker::Name.name }
+    service_name { "#{Faker::Name.name}'s Housing & Co." }
```

Comment:
Still generate a random name, but also make sure it works for names with apostrophes and ampersands.